invenio_cli/cli/packages.py

```diff
@@ -67,11 +68,28 @@ def install(cli_config, packages, skip_build):
 @pass_cli_config
 def outdated(cli_config):
     """Show outdated Python dependencies."""
-    raise click.UsageError("Not supported yet.")
+    steps = PackagesCommands.outdated_packages(packages)
+
+    on_fail = f"Failed to show outdated packages."
+    on_success = f"Outdated packages."
```

Comment:
moderate: There is no need to use an `f`-string here. These are used for formating, i.e. when you need to add a variable value in the string e.g.

```python
example = "woow"
test = f"This is a {example} test"
```

Would result in `This is a woow test`. However, in this case, there is no variables.

```suggestion
    on_fail = "Failed to show outdated packages."
    on_success = "Outdated packages."
```